profiles/common/modules/features/splash_screen/includes/splash_screen.admin.inc

```diff
@@ -32,6 +32,14 @@ function _splash_screen_settings_form() {
   // Inserts the fieldset for grouping general settings fields.
   $form['splash_screen'] = $fieldset_general;
 
+  // Splash screen title field.
+  $form['splash_screen']['splash_screen_title_value'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Splash screen title'),
+    '#description' => t('The title to use for the splash screen page'),
+    '#default_value' => variable_get('splash_screen_title_value'),
```

Comment:
It would be good to precise the default value used if the field is empty, for example: "The title to use for the splash screen page, if empty it will use the site name